sync3/connmap.go

```diff
@@ -115,9 +123,7 @@ func (m *ConnMap) Conn(cid ConnID) *Conn {
 	}
 	// e.g buffer exceeded, close it and remove it from the cache
 	logger.Info().Str("conn", cid.String()).Msg("closing connection due to dead connection (buffer full)")
-	m.mu.Lock()
```

Comment:
Presumably the point is that this lock would block because the mutex is already locked on -130.

Reply:
Bingo